GOVERNANCE.md

```diff
@@ -189,10 +214,31 @@ Example of list of contributions:
   organization
 * Other participation in the wider Node.js community
 
-The nomination passes if no collaborators oppose it after one week, and if the
-nominee publicly accepts it. In the case
-of an objection, the TSC is responsible for working with the individuals
-involved and finding a resolution.
+The nomination passes if no collaborators _explicitly_ oppose it after one week.
```

Comment:
I don't think "explicitly" helps, AFAICT the thing that needs to be clarified is whether such opposition needs to be public to count. Since we're defining below what a "valid" opposition is, let's refer to that instead.
```suggestion
The nomination passes if no collaborators oppose it (as described in the following section) after one week
```